instrument/build.go

```diff
@@ -41,15 +41,21 @@ var (
 	// BuildDate is the date this build was created.
 	BuildDate = "unknown"
 
+	// BuildDateUnixNanos is the UnixNanos since epoch representing the date this build was created.
+	BuildDateUnixNanos int64 = 0
```

Comment:
nit: `BuildTimeUnixNanos` might be more apt. Also this should probably be a const?

Reply:
sure thing, will rename.

Re `const`: nah, needs to be a `var` to allow it to be overriden from ldflags at build time.

Reply:
Ah yeah that's fair, might need to get rid of the 0 initial to silence lint.

Reply:
yep sure thing. fixed both issues.